apps/cnspec/cmd/serve_api.go

```diff
@@ -70,7 +70,13 @@ var serveApiCmd = &cobra.Command{
 			Creds:       serviceAccount,
 		}
 
+		enabledAutoUpdate := true
+		if viper.IsSet("auto_update") {
+			enabledAutoUpdate = viper.GetBool("auto_update")
+		}
+
 		scanner := scan.NewLocalScanner(
```

Comment:
why don't you just pass the auto-update config as input to `NewLocalScanner`. I can se that inside of it we create the runtime and you can specify the value in there directly.

Reply:
it seems like cnquery does things differently than cnspec. That's also what we discussed with @afiune . Why not work on making these 2 behave in the same way instead of making them differ even more

Reply:
Even if you decide to go for this approach, it seems like we use the `DefaultRuntime` here. You would be editing the default runtime which is a global variable and that's probably not what we want